### ISSUE TITLE
An error occurred while Bundler/DuplicatedGroup cop was inspecting /path/to/Gemfile.

### ISSUE THREAD

**Issue by MrJoy** [rubocop/rubocop#12162]
I know that a fix for this issue was included in 1.56.1, but... the fix is apparently not working in my case.

## Expected behavior

No error.

## Actual behavior

```
 % rubocop -d Gemfile
For /Users/jonathonfrisby/fb/google-push-listener: configuration from /Users/jonathonfrisby/fb/google-push-listener/.rubocop.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-eightyfourcodes-0.0.2/config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-eightyfourcodes-0.0.2/config/default.yml
Default configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-performance-1.19.0/config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-performance-1.19.0/config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-rake-0.6.0/config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-rake-0.6.0/config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-capybara-2.18.0/config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-capybara-2.18.0/lib/../config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-factory_bot-2.23.1/config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-factory_bot-2.23.1/lib/../config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-rspec-2.23.2/config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-rspec-2.23.2/config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-rubycw-0.1.6/config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-rubycw-0.1.6/config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-thread_safety-0.5.1/config/default.yml
configuration from #{path}
Use parallel by default.
Skipping parallel inspection: only a single file needs inspection
Inspecting 1 file
Scanning /Users/jonathonfrisby/fb/google-push-listener/Gemfile
An error occurred while Bundler/DuplicatedGroup cop was inspecting /Users/jonathonfrisby/fb/google-push-listener/Gemfile.
undefined method `value' for s(:splat,
  s(:const, nil, :LIVE_ENVS)):RuboCop::AST::Node
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cop/bundler/duplicated_group.rb:120:in `block in group_attributes'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cop/bundler/duplicated_group.rb:116:in `map'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cop/bundler/duplicated_group.rb:116:in `group_attributes'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cop/bundler/duplicated_group.rb:86:in `block in duplicated_group_nodes'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cop/bundler/duplicated_group.rb:76:in `block in group_declarations'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-ast-1.29.0/lib/rubocop/ast/node/mixin/descendence.rb:111:in `block in visit_descendants'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-ast-1.29.0/lib/rubocop/ast/node/mixin/descendence.rb:108:in `each'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-ast-1.29.0/lib/rubocop/ast/node/mixin/descendence.rb:108:in `visit_descendants'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-ast-1.29.0/lib/rubocop/ast/node/mixin/descendence.rb:112:in `block in visit_descendants'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-ast-1.29.0/lib/rubocop/ast/node/mixin/descendence.rb:108:in `each'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-ast-1.29.0/lib/rubocop/ast/node/mixin/descendence.rb:108:in `visit_descendants'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-ast-1.29.0/lib/rubocop/ast/node/mixin/descendence.rb:100:in `each_node'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cop/bundler/duplicated_group.rb:70:in `group_declarations'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cop/bundler/duplicated_group.rb:84:in `each'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cop/bundler/duplicated_group.rb:84:in `group_by'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cop/bundler/duplicated_group.rb:84:in `duplicated_group_nodes'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cop/bundler/duplicated_group.rb:71:in `on_new_investigation'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cop/commissioner.rb:160:in `block (2 levels) in invoke'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cop/commissioner.rb:171:in `with_cop_error_handling'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cop/commissioner.rb:160:in `block in invoke'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cop/commissioner.rb:160:in `each'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cop/commissioner.rb:160:in `invoke'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cop/commissioner.rb:84:in `investigate'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cop/team.rb:156:in `investigate_partial'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cop/team.rb:98:in `investigate'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/runner.rb:345:in `block in inspect_file'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/runner.rb:344:in `each'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/runner.rb:344:in `flat_map'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/runner.rb:344:in `inspect_file'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/runner.rb:287:in `block in do_inspection_loop'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/runner.rb:321:in `block in iterate_until_no_changes'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/runner.rb:314:in `loop'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/runner.rb:314:in `iterate_until_no_changes'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/runner.rb:283:in `do_inspection_loop'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/runner.rb:164:in `block in file_offenses'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/runner.rb:189:in `file_offense_cache'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/runner.rb:163:in `file_offenses'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/runner.rb:154:in `process_file'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/runner.rb:135:in `block in each_inspected_file'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/runner.rb:134:in `each'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/runner.rb:134:in `reduce'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/runner.rb:134:in `each_inspected_file'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/runner.rb:120:in `inspect_files'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/runner.rb:73:in `run'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cli/command/execute_runner.rb:26:in `block in execute_runner'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cli/command/execute_runner.rb:52:in `with_redirect'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cli/command/execute_runner.rb:25:in `execute_runner'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cli/command/execute_runner.rb:17:in `run'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cli/command.rb:11:in `run'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cli/environment.rb:18:in `run'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cli.rb:118:in `run_command'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cli.rb:125:in `execute_runners'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cli.rb:51:in `block in run'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cli.rb:77:in `profile_if_needed'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/lib/rubocop/cli.rb:43:in `run'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/exe/rubocop:19:in `block in <top (required)>'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/lib/ruby/3.2.0/benchmark.rb:311:in `realtime'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/gems/rubocop-1.56.1/exe/rubocop:19:in `<top (required)>'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/bin/rubocop:25:in `load'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fb-google-push-listener/bin/rubocop:25:in `<main>'
.

1 file inspected, no offenses detected

1 error occurred:
An error occurred while Bundler/DuplicatedGroup cop was inspecting /Users/jonathonfrisby/fb/google-push-listener/Gemfile.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
https://github.com/rubocop/rubocop/issues

Mention the following information in the issue report:
1.56.1 (using Parser 3.2.2.3, rubocop-ast 1.29.0, running on ruby 3.2.2) [arm64-darwin21]
Finished in 0.3840370001271367 seconds
```

## Steps to reproduce the problem

This construction in Gemfile will do the trick:

```ruby
group(*LIVE_ENVS) do
  gem "sentry-ruby",    "~> 5.0"
  gem "sentry-sidekiq", "~> 5.0"
end
```

## RuboCop version

```
1.56.1 (using Parser 3.2.2.3, rubocop-ast 1.29.0, running on ruby 3.2.2) [arm64-darwin21]
  - rubocop-performance 1.19.0
  - rubocop-rake 0.6.0
  - rubocop-rspec 2.23.2
  - rubocop-rubycw 0.1.6
  - rubocop-thread_safety 0.5.1
```

**Comment by MrJoy** [comment]
Thank you!

**Comment by emm26** [comment]
Hi, we're still seeing the same issues as of v1.56.2:
```
$ bundle exec rubocop -d Gemfile
For /Users/edgarm/Documents/GitHub/project: configuration from /Users/edgarm/Documents/GitHub/project/.rubocop.yml
Inheriting configuration from /Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/bundler/gems/style-guide-9f5af85a4340/config/rubocop.yml
configuration from /Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-faker-1.1.0/config/default.yml
configuration from /Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-faker-1.1.0/config/default.yml
Default configuration from /Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/config/default.yml
configuration from /Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-graphql-1.4.0/config/default.yml
configuration from #{path}
configuration from /Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-performance-1.19.0/config/default.yml
configuration from /Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-performance-1.19.0/config/default.yml
configuration from /Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-rails-2.20.2/config/default.yml
configuration from /Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-rails-2.20.2/config/default.yml
configuration from /Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-rake-0.6.0/config/default.yml
configuration from /Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-rake-0.6.0/config/default.yml
configuration from /Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-capybara-2.18.0/config/default.yml
configuration from /Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-capybara-2.18.0/lib/../config/default.yml
configuration from /Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-factory_bot-2.23.1/config/default.yml
configuration from /Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-factory_bot-2.23.1/lib/../config/default.yml
configuration from /Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-rspec-2.23.2/config/default.yml
configuration from /Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-rspec-2.23.2/config/default.yml
Use parallel by default.
Skipping parallel inspection: only a single file needs inspection
Inspecting 1 file
Scanning /Users/edgarm/Documents/GitHub/project/Gemfile
An error occurred while Bundler/DuplicatedGroup cop was inspecting /Users/edgarm/Documents/GitHub/project/Gemfile.
undefined method `value' for s(:splat,
  s(:const, nil, :GROUP_ELEMS)):RuboCop::AST::Node

              argument.value.to_s
                      ^^^^^^
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cop/bundler/duplicated_group.rb:120:in `block in group_attributes'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cop/bundler/duplicated_group.rb:116:in `map'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cop/bundler/duplicated_group.rb:116:in `group_attributes'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cop/bundler/duplicated_group.rb:86:in `block in duplicated_group_nodes'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cop/bundler/duplicated_group.rb:76:in `block in group_declarations'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-ast-1.29.0/lib/rubocop/ast/node/mixin/descendence.rb:111:in `block in visit_descendants'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-ast-1.29.0/lib/rubocop/ast/node/mixin/descendence.rb:108:in `each'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-ast-1.29.0/lib/rubocop/ast/node/mixin/descendence.rb:108:in `visit_descendants'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-ast-1.29.0/lib/rubocop/ast/node/mixin/descendence.rb:112:in `block in visit_descendants'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-ast-1.29.0/lib/rubocop/ast/node/mixin/descendence.rb:108:in `each'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-ast-1.29.0/lib/rubocop/ast/node/mixin/descendence.rb:108:in `visit_descendants'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-ast-1.29.0/lib/rubocop/ast/node/mixin/descendence.rb:100:in `each_node'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cop/bundler/duplicated_group.rb:70:in `group_declarations'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cop/bundler/duplicated_group.rb:84:in `each'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cop/bundler/duplicated_group.rb:84:in `group_by'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cop/bundler/duplicated_group.rb:84:in `duplicated_group_nodes'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cop/bundler/duplicated_group.rb:71:in `on_new_investigation'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cop/commissioner.rb:160:in `block (2 levels) in invoke'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cop/commissioner.rb:171:in `with_cop_error_handling'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cop/commissioner.rb:160:in `block in invoke'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cop/commissioner.rb:160:in `each'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cop/commissioner.rb:160:in `invoke'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cop/commissioner.rb:84:in `investigate'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cop/team.rb:156:in `investigate_partial'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cop/team.rb:98:in `investigate'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/runner.rb:345:in `block in inspect_file'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/runner.rb:344:in `each'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/runner.rb:344:in `flat_map'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/runner.rb:344:in `inspect_file'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/runner.rb:287:in `block in do_inspection_loop'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/runner.rb:321:in `block in iterate_until_no_changes'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/runner.rb:314:in `loop'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/runner.rb:314:in `iterate_until_no_changes'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/runner.rb:283:in `do_inspection_loop'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/runner.rb:164:in `block in file_offenses'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/runner.rb:189:in `file_offense_cache'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/runner.rb:163:in `file_offenses'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/runner.rb:154:in `process_file'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/runner.rb:135:in `block in each_inspected_file'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/runner.rb:134:in `each'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/runner.rb:134:in `reduce'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/runner.rb:134:in `each_inspected_file'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/runner.rb:120:in `inspect_files'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/runner.rb:73:in `run'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cli/command/execute_runner.rb:26:in `block in execute_runner'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cli/command/execute_runner.rb:52:in `with_redirect'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cli/command/execute_runner.rb:25:in `execute_runner'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cli/command/execute_runner.rb:17:in `run'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cli/command.rb:11:in `run'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cli/environment.rb:18:in `run'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cli.rb:118:in `run_command'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cli.rb:125:in `execute_runners'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cli.rb:51:in `block in run'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cli.rb:77:in `profile_if_needed'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/lib/rubocop/cli.rb:43:in `run'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/exe/rubocop:19:in `block in <top (required)>'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/3.1.0/benchmark.rb:311:in `realtime'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rubocop-1.56.2/exe/rubocop:19:in `<top (required)>'
/Users/edgarm/.rbenv/versions/3.1.2/bin/rubocop:25:in `load'
/Users/edgarm/.rbenv/versions/3.1.2/bin/rubocop:25:in `<top (required)>'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.24/lib/bundler/cli/exec.rb:58:in `load'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.24/lib/bundler/cli/exec.rb:58:in `kernel_load'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.24/lib/bundler/cli/exec.rb:23:in `run'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.24/lib/bundler/cli.rb:486:in `exec'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.24/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.24/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.24/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.24/lib/bundler/cli.rb:31:in `dispatch'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.24/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.24/lib/bundler/cli.rb:25:in `start'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.24/exe/bundle:48:in `block in <top (required)>'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.24/lib/bundler/friendly_errors.rb:120:in `with_friendly_errors'
/Users/edgarm/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.24/exe/bundle:36:in `<top (required)>'
/Users/edgarm/.rbenv/versions/3.1.2/bin/bundle:25:in `load'
/Users/edgarm/.rbenv/versions/3.1.2/bin/bundle:25:in `<main>'
.

1 file inspected, no offenses detected

1 error occurred:
An error occurred while Bundler/DuplicatedGroup cop was inspecting /Users/edgarm/Documents/GitHub/project/Gemfile.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
https://github.com/rubocop/rubocop/issues

Mention the following information in the issue report:
1.56.2 (using Parser 3.2.2.3, rubocop-ast 1.29.0, running on ruby 3.1.2) [x86_64-darwin22]
Finished in 0.8935340000025462 seconds 
```

### Gemfile
```
GROUP_ELEMS = %i(
   first_elem
   other_elem
)

group(*GROUP_ELEMS) do
  gem "sentry-ruby",    "~> 5.0"
  gem "sentry-sidekiq", "~> 5.0"
end

```

**Comment by koic** [comment]
@emm26 The patch #12163 resolving this issue has not yet been released. Please wait for the release following 1.56.2.

**Comment by emm26** [comment]
Thank you, much appreciated!
